website/content/docs/plugins/containerized-plugins.mdx

```diff
@@ -0,0 +1,253 @@
+---
+layout: docs
+page_title: Containerized Plugins (Beta)
+description: Running external plugins for Vault in containers.
+---
+
+# Containerized Plugins (Beta)
+
+If running on Linux, Vault can optionally run external plugins in containers.
+This provides increased levels of isolation between the plugin and Vault, as
+well as the plugin and other plugins, compared to the default behavior of
+running plugins as a sub-process with the same user and environment variables
+as Vault.
+
+## Prerequisites
+
+* Linux. Containerized plugin support is currently limited to Linux.
+
+* Local Docker Engine API available to Vault. Vault uses the
+  [Docker SDK](https://pkg.go.dev/github.com/docker/docker) to manage containerized
+  plugins, which means it needs access to a Docker Engine API. This could be
+  backed by Docker, Podman, or any other Docker-compatible container engines.
+  Use the `DOCKER_HOST` environment variable to configure a custom address.
+
+* (Optional) gVisor installed and configured as a runtime. Vault uses gVisor's
+  runsc container runtime by default. However, it can also be configured to use
+  other runtimes such as Docker's default, runc. In addition to the
+  [official installation instructions](https://gvisor.dev/docs/user_guide/install/),
+  runsc requires an additional runtime argument to enable Unix domain socket
```

Comment:
```suggestion
  `runsc` container runtime by default. However, it can also be configured to use
  other runtimes such as Docker's default, `runc`. In addition to the
  [official installation instructions](https://gvisor.dev/docs/user_guide/install/),
  `runsc` requires an additional runtime argument to enable Unix domain socket
```